src/tools/std20/views/test/transform.test.cpp

```diff
@@ -0,0 +1,132 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/views.hpp"
+
+// other includes
+#include <forward_list>
+#include <list>
+#include <vector>
+#include "tools/std20/algorithm.hpp"
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "subrange" ) {
+
+  const std::vector< int > equal = { -2, -1, 0, 1, 2 };
+
+  auto transform = [] ( auto&& value ) { return value - 2; };
+
+  GIVEN( "a container with forward iterators" ) {
+
+    std::forward_list< int > values = { 0, 1, 2, 3, 4 };
+
+    WHEN( "when the container and the transformation are used" ) {
+
+      auto chunk = values | std20::views::transform( transform );
+      using Range = decltype(chunk);
+
+      THEN( "the subrange satisfies the required consepcts" ) {
```

Comment:
spelling "concepts", indentation (, changing then label mention "transform"?)

Likewise for subsequent THENs within this test